### PR TITLE
fix(DocFindBar): Add aria-label to docfindbar input field

### PR DESCRIPTION
**Pull request by fsuarez-globant**
fix(DocFindBar): Add aria-label to docfindbar input field

### CODE REVIEW

File: src/lib/viewers/doc/DocFindBar.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ class DocFindBar extends EventEmitter {
         // Find input field
         this.findFieldEl = document.createElement('input');
         this.findFieldEl.classList.add('bp-doc-find-field');
+        this.findFieldEl.setAttribute('aria-label', 'Search input');
```
Comment (jstoffan) [inline src/lib/viewers/doc/DocFindBar.js:85]: Hi @fsuarez-globant, thanks for submitting this change. We require all our user-facing strings to support internationalization. Can you please add this string to `src/i18n/en-US.properties` and use it via the `__` helper?

Comment (fsuarez-globant) [reply src/lib/viewers/doc/DocFindBar.js:85]: @jstoffan of course, I'll add this string in the internationalization file.

File: src/i18n/en-US.properties [hunk 2]
```diff
@@ -54,6 +54,8 @@ find_next=Find next
 find_previous=Find previous
 # Button tooltip for closing the find bar
 find_close=Close
+# Search input Aria-label attribute
+search_input=Search input
```
Comment (jstoffan) [inline src/i18n/en-US.properties:58]: Can we use `find_label` to stay consistent with the other `find_` keys? That will keep them all together in the output JSON file, as well.

Comment (fsuarez-globant) [reply src/i18n/en-US.properties:58]: @jstoffan I'll change this as well

**Review by jstoffan — APPROVED** [review]

**Pull request merged**